german/Kopfsprung/Kopfsprung.md

```diff
@@ -6,4 +6,5 @@ Das Riff ist mit wunderschönen Korallen bewachesen, welche eine bunte Schar Fis
 Werden Sie:
 
 [In das Boot zurückkehren?](../ruderboot.md)
+[Die Korallen nochmal genauer ansehen?](../Korallen/Korallen.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.